esmvalcore/config-developer.yml

```diff
@@ -27,6 +27,9 @@ CMIP6:
     ETHZ: '{exp}/{mip}/{short_name}/{dataset}/{ensemble}/{grid}/'
   input_file: '{short_name}_{mip}_{dataset}_{exp}_{ensemble}_{grid}*.nc'
   output_file: '{project}_{dataset}_{mip}_{exp}_{ensemble}_{short_name}_{start_year}-{end_year}'
+  institutes:
+    'CESM2-WACCM': ['NCAR']
```

Comment:
This should not be needed if we just update the cmor tables to the latest version. I made a pull request here: https://github.com/ESMValGroup/ESMValCore/pull/363

Reply:
So this list will have to be updated every time the tables get updated?

Reply:
I think the other institute can also be removed.

Reply:
careful what you merge first - if you merge this PR before #363 then it won't work proper :beer: